app/posts/publish-teacher-training-courses/2023-04-20-removing-course-vacancies.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Removing course vacancies
+description: We removed course vacancies as this was misleading for candidates
+date: 2023-04-20
+tags:
+  - school placements
+  - vacancies
+related:
+  items:
+    - text: Removing the school placement vacancy status
+      href: /find-teacher-training/removing-the-school-placement-vacancy-status/
+    - text: Updating course vacancies
+      href: /publish-teacher-training-courses/updating-course-vacancies/
+screenshots:
+  items:
+    - text: Courses list
+      src: courses-list.png
+    - text: Course details with ‘Close course’ button
+      src: course-details--close.png
+    - text: Close course
+      src: close-course.png
+    - text: Course closed
+      src: course-closed.png
+    - text: Course details with ‘Open course’ button
+      src: course-details--open.png
+    - text: Open course
+      src: open-course.png
+    - text: Course opened
+      src: course-opened.png
+    - text: Withdraw open course
+      src: withdraw-open-course.png
+    - text: Withdraw closed course
+      src: withdraw-closed-course.png
+    - text: Training partners’ courses list
+      src: courses-list--training-partners.png
+---
+
+When a provider publishes a course, they can list all the schools where a candidate can gain experience.
+
+Previously, once a course was published, it was open to applications. This meant there were vacancies at some or all of the schools listed.
+
+We provided a way for users to state whether each school had vacancies.
+
+If the provider stated there were no vacancies for all schools on the list, the course is closed.  Closed courses can still be found on Find postgraduate teacher training (Find), but a candidate cannot apply.
+
+We have removed the link between the ‘Open’ and ‘Closed’ course statuses and vacancies. Users can simply state whether the course is open or closed.
```

Comment:
Wondering why the first is `‘Open’` and `‘Closed’` and the second `open` and `closed`?

Reply:
The way Simon has written it is correct. I'll explain why via Slack :)